### DISCUSSION TITLE
Auto-save 2 days?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6842] [Ideas]
How about having a feature to auto-save last two days? I mean, most of the time, when something bad happens and I want to revert, autosave is too late and I have to restart from the beginning.

**Comment by LeHerosInconnu** [comment]
Hello @HubKing,

Maybe something similar to this proposal: https://github.com/ihhub/fheroes2/issues/1529 could also be applied to the AUTOSAVE files. :)
So: "User can decide to have multiple AUTOSAVE files, for example 3, 5 or 10."
